test/UnitTest/Microsoft.AspNet.OData.Test.Shared/OpenComplexTypeTest.cs

```diff
@@ -603,17 +603,6 @@ public ITestActionResult PatchToAddress(int key, Delta<OpenAddress> address)
                     // Nested complex property
                     Assert.NotNull(lineA.PhoneInfo);
                     Assert.Equal(7654321, lineA.PhoneInfo.PhoneNumber);
-
-                    object nestedLineAValue;
-                    // Fetch LineA property using TryGetNestedPropertyValue
-                    Assert.True(address.TryGetNestedPropertyValue("LineA", out nestedLineAValue));
-                    Delta<LineDetails> deltaLineA = nestedLineAValue as Delta<LineDetails>;
-                    Assert.NotNull(deltaLineA);
-
-                    // Nested complex property
-                    dynamic nestedLineA = deltaLineA;
-                    Assert.NotNull(nestedLineA.PhoneInfo);
-                    Assert.Equal(7654321, nestedLineA.PhoneInfo.PhoneNumber);
```

Comment:
Curious about this deletion

Reply:
This method `TryGetNestedPropertyValue` had been removed from `DeltaOfTStructuralType` because it was only being called in this test class. Have added it back as it really has no effect on anything and returned this code segment.